dev-tools/ansible/roles/pga/templates/pga-ssl-vhost.conf.j2

```diff
@@ -1,4 +1,4 @@
-<VirtualHost *:80>
+<VirtualHost *:8008>
```

Comment:
Shouldn't these files use the `pga_default_http_port` variables? See also line 13 and pga-vhost.conf.j2.